libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -1650,7 +1662,17 @@ MAV_RESULT GCS_MAVLINK::handle_preflight_reboot(const mavlink_command_long_t &pa
             }
 #endif
         }
-
+        if (should_zero_rc_outputs_on_reboot()) {
+            reboot_zero_rc_outputs();
```

Comment:
humn... the method should be `zero_rc_outputs()` or the `should_zero_rc_outputs_on_reboot()` should be moved inside the method. Otherwise it is redundant

Reply:
Renamed it.